backend/src/main/resources/application-dev.properties

```diff
@@ -9,20 +9,16 @@ server.ssl.key-store=classpath:certs/dev.money-tree.tech/cert.p12
 server.ssl.key-store-password= ${SERVER_SSL_KEYSTOREPASSWORD}
 server.ssl.keyStoreType= PKCS12
 
-spring.data.neo4j.uri=bolt://dev.money-tree.tech:7687
+spring.data.neo4j.uri=neo4j://dev.money-tree.tech:7687
```

Comment:
Why was this change ? is it because there is a service called neo4j in docker compose ?

Reply:
Yeah exactly. And for some reasons when I was making a call to the prod environment, it was creating a user in the dev env.